### ISSUE TITLE
Window not defined in node environment

### ISSUE THREAD

**Issue by nmfzone** [just-jeb/angular-builders#947]
When I want to run jest in node environment (E2E with Puppeteer), it gives me an error:

```
ReferenceError: window is not defined

      at Object.<anonymous> (node_modules/@angular-builders/jest/src/jest-config/global-mocks.ts:13:23)
```

Is it possible to remove the default `setupFilesAfterEnv` config by `@angular-builders/jest` ?

## Environment

<pre>
<code>
Libs
- @angular/core version: 8.2.1
- @angular-devkit/build-angular version: 0.803.1
- @angular-builders/jest version: 8.3.2

- Node version: 14.15.5
- Platform: Mac
</code>
</pre>

**Comment by just-jeb** [comment]
I'm not entirely sure how you get to this flow... As you can see from the [example](https://github.com/just-jeb/angular-builders/blob/master/packages/jest/examples/multiple-apps/angular.json#L205) the builder that runs e2e tests is @angular-devkit/build-angular:protractor, and has nothing to do with @angular-builders/jest builder. The [command](https://github.com/just-jeb/angular-builders/blob/master/packages/jest/examples/multiple-apps/package.json#L10) that runs `e2e` tests is `ng e2e`, not `ng test`.   
So I how do you get to `node_modules/@angular-builders/jest/src/jest-config/global-mocks.ts` in your `e2e` tests?

**Comment by nmfzone** [comment]
@just-jeb For me, I don't use Protractor at all. I've used jest here for E2E. That's why I've got that error.

**Comment by nmfzone** [comment]
@just-jeb I just want to let you know, the [jest-preset-angular](https://github.com/thymikee/jest-preset-angular) don't place the [setup-jest.js](https://github.com/thymikee/jest-preset-angular/blob/master/src/config/setup-jest.ts) in their [default config](https://github.com/thymikee/jest-preset-angular/blob/master/presets/index.js). We need to put their `setup-jest.js` in the `jest.config.js` by our self. That's why we (as a user) have a possibility to not use their setup.

**Comment by nmfzone** [comment]
How can I (as a user), change window to global in `global-mocks`?

**Comment by just-jeb** [comment]
As a user you can't, I'm talking about changing it in the builder.

**Comment by nmfzone** [comment]
@just-jeb I'm not sure, since `global` will not available in `jsdom` environment, right?
Maybe you can give an option, to load the `global-mock` or not. Or maybe, just check if `window` is available, then use the `global-mocks`.

**Comment by just-jeb** [comment]
Seems like providing an option for disabling `global-mocks` would solve both this and #696

**Comment by nmfzone** [comment]
@just-jeb Maybe, yes..

**Comment by just-jeb** [comment]
The new `globalMocks` option is available in `v11.2.0-beta.0`, please confirm it works for you.  
For more details refer to [documentation](https://github.com/just-jeb/angular-builders/tree/master/packages/jest#builder-options).

**Comment by nmfzone** [comment]
@just-jeb Awesome 👌🏻 But I think I can't test it currently. I'll confirm if I have tested it, soon.
